terrain_map:
  # Defining the search areas
  search_areas:
    - centre_front_1
#    - centre_front_2
#    - centre_back
#    - left_lateral
#    - right_lateral
  centre_front_1: {min_x: -0., max_x: 1.6, min_y: -0.5, max_y: 0.5, min_z: -1.2, max_z: 0., resolution: 0.02}
#  centre_front_2: {min_x: 3.0, max_x: 3.5, min_y: -0.85, max_y: 0.85, min_z: -1.2, max_z: 0., resolution: 0.08}
#  centre_back: {min_x: -0.75, max_x: -0.5, min_y: -0.85, max_y: 0.85, min_z: -1.2, max_z: 0., resolution: 0.08}
#  left_lateral: {min_x: -0.75, max_x: 5., min_y: -1.25, max_y: 0.85, min_z: -1.2, max_z: 0., resolution: 0.04}
#  right_lateral: {min_x: -0.75, max_x: 5., min_y: 0.85, max_y: 1.25, min_z: -1.2, max_z: 0., resolution: 0.04}
  
  # Defining the interest region for costmap generation
  interest_region:
    radius_x: 1.5
    radius_y: 5.5
  
  # Defining the features for the costmap generation
  features:
    slope: {enable: false, weight: 1}
    height_deviation: {enable: true, weight: 1, neighboring_area: {square_size: 0.12, resolution: 0.02},
     flat_height_deviation: 0.01, max_height_deviation: 0.06, min_allowed_height: -0.10}
    curvature: {enable: false, weight: 1}
